### ISSUE TITLE
Submit site has a case of low contrast

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wporg-showcase-2022#165]
When you go to Submit a site, https://wordpress.org/showcase-v2/submit-a-wordpress-site/, the "(required)" is a bit low contrast.

We can either show it using the Charcoal 4 spot color instead, that should have sufficient contrast.

Not urgent.

**Comment by ryelle** [comment]
Weird, it had passing contrast on my local site.
<img width="470" alt="Screenshot 2023-09-21 at 11 35 30 AM" src="https://github.com/WordPress/wporg-showcase-2022/assets/541093/44ee8ff5-8cdb-4917-9d61-1bb253bb3626">

Looks like there's an opacity on it that should be removed.
